docs/cpp/config-mingw.md

```diff
@@ -425,6 +425,16 @@ If you have Visual Studio or WSL installed, you may need to change `compilerPath
 
 You must follow the steps on the [MSYS2 website](https://www.msys2.org/) and use the MSYS CLI to install MinGW-w64, which contains those tools. You will also need to install the full MinGW-w64 toolchain (`pacman -S --needed base-devel mingw-w64-x86_64-toolchain`) to get the `gdb` debugger.
```

Comment:
Should the package name be updated for this section as well?

Reply:
Yes, I updated it